shell/store/type-map.utils.ts

```diff
@@ -123,6 +123,10 @@ export function createHeaders(
  * Given a schema's attribute.column value create a header
  */
 export function headerFromSchemaColString(colName: string, schema: Schema, rootGetters: VuexStoreGetters, pagination: boolean, ageColumn: TableColumn): TableColumn {
+  if (!schema) {
+    throw new Error(`Schema is null`);
```

Comment:
If we're adding dedicated error handling the description could be a little more

```suggestion
    throw new Error(`Unable to create header for column '${colName}' from schema: schema is missing`);
```